groclient/client.py

```diff
@@ -698,6 +698,60 @@ def get_geojson(self, region_id, zoom_level=7):
         """
         return lib.get_geojson(self.access_token, self.api_host, region_id, zoom_level)
 
+    def get_descendant(
+        self,
+        entity_type,
+        entity_id,
+        distance=None,
+        include_details=True,
+    ):
+        """Given an item/metric/region id return all ids contained in the graph, and their relative details
```

Comment:
"all ids contained in the graph" is incorrect, sounds like we're returning the entire graph.  How about "given an item, metric or region, returns all its descendants i.e. entities that are "contained" in the given entity".